### DISCUSSION TITLE
Google AMP errors with Google Sitekit(Wordpress)

### DISCUSSION THREAD

**Discussion by MrDevLovePer** [Automattic/amp-wp#6377] [General]
After checking several times I noticed that using the **Google AMP** plugin Along with the **google Sitekit** plugin is causing huge errors in Page Insights.
Anyone, please check this issue and confirm whether this true or not?

**Comment by westonruter** [comment]
What errors are you seeing caused by the combination of AMP and Site Kit?

**Reply by gencial** [reply]
How can I display the comments section in the amp?
Does the amp plugin support the comments section?
If the section supports comments, how can I display them?

**Comment by westonruter** [comment]
Please submit your question to the plugin's support forum: https://wordpress.org/support/plugin/amp/
